scripts/copy_meta_from_json.py

```diff
@@ -62,6 +81,7 @@ def embed_metadata(game_metadata, test_results):
     with open(f"./games/{game_metadata["filename"]}.js", "w") as f:
```

Comment:
the game metadata filename should be accessed using single quotes here as it's withing a double-quoted f-string